tests/auth-react/flask-server/app.py

```diff
@@ -17,6 +17,8 @@
 from dotenv import load_dotenv
 from flask import Flask, g, jsonify, make_response, request
 from flask_cors import CORS
+from typing_extensions import Literal
```

Comment:
Reset unrelated changes in this file

Reply:
`dashboard/__iniy__.py` doesn't globally import `dashboard/recipe.py` so these imports change